### ISSUE TITLE
Save words with specific reading to the vocab decks

### ISSUE THREAD

**Issue by Chiara3589** [SYtor/Kanji-Dojo#225]
I found that even if I save a word, it's going to be saved as the app wants and not how I searched for it.
Fro example I searched for 誰 but when I do my flash cards it's saved as た, or I want to save the word with kanjis but it's saved in hiragana when doing my flashcards

**Comment by syt0r** [comment]
Yes, this is intended behavior for now but I plan to change it. I'm going to remove reading priority configuration which controls this behavior so that the vocab decks would track only one word reading per card

**Comment by Mitsuha420** [comment]
Been struggling with this as well. It's always a disappointment whenever I save a new word, and it ends up showing up in hiragana during reviews. Makes saving that word quite pointless unfortunately, at least for my needs.

As far as my experience goes, there's also no way to know in advance which way a word will show up during practice, at least I couldn't find any logic behind it. I think it's a pretty big issue in an otherwise amazing app, so it would be awesome to have this functionality updated. Thank you.

**Comment by syt0r** [comment]
Working on this right now, hopefully will be available this month 🙏

**Comment by syt0r** [comment]
Fixed in v2.1.6
